pkg/GAPJulia/JuliaInterface/src/JuliaInterface.c

```diff
@@ -332,6 +333,38 @@ static Obj FuncJuliaGetFieldOfObject(Obj self, Obj super_obj, Obj field_name)
 
 static Obj IsOutputStream;
 
+static Obj FuncSTREAM_DISPLAYOBJ(Obj self, Obj stream, Obj obj)
+{
+    syJmp_buf readJmpError;
+
+    if (CALL_1ARGS(IsOutputStream, stream) != True) {
+        ErrorQuit("STREAM_DISPLAYOBJ: <outstream> must be an output stream", 0,
```

Comment:
Please `clang-format` the file, and/or manually adjust the formatting (see <https://travis-ci.com/github/oscar-system/GAP.jl/jobs/300342903>).